openapi.yaml

```diff
@@ -2851,6 +2851,20 @@ components:
           $ref: '#/components/schemas/period-start'
         values:
           $ref: '#/components/schemas/limit-value-set'
+        conditional-values:
```

Comment:
MISO is ostensibly switching from conditional ratings to topology-based ratings, but we may yet have need of this, too. I mention that because we thought we might need an indication of which condition was in the active topology.

Reply:
So... I wonder if this is the same thing.  Whether this is "topological" or not may be a bit of an academic debate.  I think I should have put more in the comments here.  If you look at what I put in network-segment, this refers to altering-conditions, which:

A condition on a given segment that may have specialized ratings against it.  
This is typically used for underground cables that are next to each other in a tray.  
If certain cables are switched on or off, it significantly changes the rating of the line (segment).  

This is, as I understand it, something that ISO-NE needs specifically to manage underwater lines.  I don't know if the EMS (NETMOM in GE's case obviously) actually recognizes these conditions as different topology.  Rather, since there are a bunch of cables next to each other in a tray, having different combinations of them turn/on-off is actually quite complex, as they can heat each other up.  The way it's been explained to me, it's more like arbitrary configuration options for the line that happen to update the limit.  This is a bit different than having a device in NETMOM become a limiting device due to topology.  

Is there a relationship between these things?  Maybe, I don't know.  Does MISO have a similar need?

Reply:
We *currently* don't utilize our EMS capability to drive ratings by allowing the breaker status to drive which segments are used to determine the overall facility ratings, but that's supposed to be changing. Today, we model "conditional ratings" for this purpose and use a different conditional rating based on the breaker status in RT. This custom is limited to changes based on one or two breakers' status, IIRC. In any event we plan to retire this custom, but we've hit some limitations in our testing that have me hedging.

Regardless, I'm primarily looking at this from the general TROLIE user perspective, and I think an indication as to which condition is assumed in the submission/proposal is valuable.

The broader discussion of conditional vs topological ratings is better suited to a call.